llvm/test/MC/Disassembler/X86/apx/sha1msg1.txt

```diff
@@ -0,0 +1,10 @@
+# RUN: llvm-mc --disassemble %s -triple=x86_64 | FileCheck %s --check-prefixes=ATT
```

Comment:
Should be better to use file/cpuid instead of file/instruction to organize the test?

Reply:
I have a justification... 

We used to organize the test by cpuid. But the folder name `apx` here is the already about  cpuid, and we have so many instructions for apx, if we put it into a test file, it would be very huge.